[core]/es_extended/client/modules/points.lua

```diff
@@ -0,0 +1,60 @@
+local Markers = {}
+
+---@func fun(coords: vector3, radius: number, data?: any)
+function ESX.CreateMarker(data) 
+    local Index = #Points+1
+    Markers[Index] = data
+    return Markers[Index]
+end
+
+CreateThread(function()
+    while true do 
+        Wait(0)
+        local InPoint = false
+        local Coords = GetEntityCoords(ESX.PlayerData.Ped)
+        for i = 1, #Markers do
+            local Marker = Markers[i]
+            Marker.distance = #(Coords - Marker.Coords) 
+            if Marker.distance < Marker.radius then 
+                if Marker.status and Marker.Inside then 
```

Comment:
Marker.Inside, Marker.inside bad variable name, and don't use. You dont't set Marker.inside.

Reply:
Fixed Marker.Inside is now a check if the function exists